libs/net/dal/Migrations/20230921212403_1.0.69.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using TNO.DAL;
+
+#nullable disable
+
+namespace TNO.DAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class _1069 : SeedMigration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.AddColumn<string>(
+                name: "aliases",
+                table: "contributor",
+                type: "text",
```

Comment:
You should update the `ContributorConfiguration` to include your new properties and also should limit aliases to 500 max length.